lib/doorkeeper/oauth/refresh_token_request.rb

```diff
@@ -99,7 +99,7 @@ def validate_client_match
 
       def validate_scope
         if @original_scopes.present?
-          ScopeChecker.valid?(@original_scopes, refresh_token.scopes)
+          ScopeChecker.valid?(nil, @original_scopes, refresh_token.scopes)
```

Comment:
Also using kwargs will avoid such cases when we need to pass `nil` values as a first argument. And we can use default values